jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/main/java/org/jbpm/workbench/es/client/editors/jobdetails/JobParameterListViewImpl.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jbpm.workbench.es.client.editors.jobdetails;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Function;
+import javax.annotation.PostConstruct;
+import javax.enterprise.context.Dependent;
+
+import com.google.gwt.cell.client.TextCell;
+
+import com.google.gwt.user.cellview.client.Column;
+import com.google.gwt.user.client.ui.IsWidget;
+import com.google.gwt.user.client.ui.Widget;
+import com.google.gwt.view.client.ListDataProvider;
+import org.jbpm.workbench.es.client.i18n.Constants;
+import org.jbpm.workbench.es.model.RequestParameterSummary;
+
+import org.uberfire.ext.widgets.common.client.tables.SimpleTable;
+import org.uberfire.ext.widgets.table.client.ColumnMeta;
+
+@Dependent
+public class JobParameterListViewImpl implements IsWidget {
+
+    public static final String COL_PARAM_ID = "ParamId";
+    public static final String COL_PARAM_VALUE = "ParamValue";
+    final Constants constants = Constants.INSTANCE;
+    protected SimpleTable<RequestParameterSummary> parameterSummarySimpleTable;
```

Comment:
@nmirasch would it be reasonable to consider updating this implementation to use Errai instead? Something towards what we did in here: https://github.com/kiegroup/jbpm-wb/blob/a434fbd7440c2d81c9eb47dfb0a5037cd8f5ba92/jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/newcase/NewCaseInstanceViewImpl.java#L107